proto/public/api.proto

```diff
@@ -173,8 +173,13 @@ message StatTable {
       Resource resource = 1;
       string time_window = 2;
 
+      // number of pods in this resource that have conduit injected
       uint64 meshed_pod_count = 3;
+      // number of pods in this resource
       uint64 total_pod_count = 4;
+      // number of pods in this resource that have Phase PodFailed
+      uint64 errored_pod_count = 6;
```

Comment:
nitpick on naming, as this is intended to communicate the number of pods in the `PodFailed` state, how about just calling it `failed_pod_count`?

Reply:
yep, that's a good idea!